cves/CVE-2010-1452.yml

```diff
@@ -80,9 +85,9 @@ unit_tested:
 
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  answer:
-  code:
-  fix:
+  answer: No unit tests existed for the affected modules.
```

Comment:
Were there different types of tests or any types of tests at all? For example were there integration tests that could have mitigated the problem?

Reply:
There were informal test scripts in the repo, but they did not touch the affected modules. Will update to reflect that.